package.json

```diff
@@ -509,33 +631,24 @@
           "default": [],
           "description": "The Additional Config Arguments, .e.g [\"--cc=gcc\", \"--myopt=xxx\"]"
         },
-        "xmake.runningTargetsArguments": {
-          "type": "object",
-          "default": {
-            "default": []
-          },
-          "description": "The Running Targets Arguments, .e.g {\"targetName\": [\"args\", \"...\"]}"
-        },
-        "xmake.debuggingTargetsArguments": {
-          "type": "object",
-          "default": {
-            "default": []
-          },
-          "description": "The Debugging Targets Arguments, .e.g {\"targetName\": [\"args\", \"...\"]}"
+        "xmake.envBehaviour": {
+          "type": "string",
+          "default":"merge",
+          "enum": ["merge", "override"],
+          "description": "Environment behaviour between launch.json and xmake envs", 
+          "enumDescriptions": [
+            "This will concat launch.json envs and xmake envs",
+            "Launch configurations will override xmake envs"
+          ]
         },
-        "xmake.debugConfigType": {
+        "xmake.debuggerBackend": {
```

Comment:
and here, This time breaking will the user's existing configuration.

Reply:
Ok, I revert back to the old one